test/testSetup.ts

```diff
@@ -252,6 +289,11 @@ export class TestSetup {
     utilities.log(`${this.testSuiteSuffixName} - selecting SFDX: Set a Default Org...`);
     const inputBox = await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Set a Default Org', 1);
 
+    // Type the scratch org's name into the filter. Do this in case the
```

Comment:
With the additional test suites, the org list is now too long to pick from.  I added code to type in the name of the org's alias.  This will filter the results in the picklist and make the target org visible in the list.